libautoscoper/src/Tracker.cpp

```diff
@@ -585,7 +599,21 @@ void Tracker::optimize(int frame, int dFrame, int repeats, int opt_method, unsig
 
 // Calculate Correlation for Bone Matching
 std::vector <double> Tracker::trackFrame(unsigned int volumeID, double* xyzypr) const
-  {
+{
+
+    // Upon a call to the Tracker::trackFrame() function, the following steps are performed:
+    //
+    // 1) Given a new pose, for each camera view:
+    //  a) Compute the modelView and inverseModelView matrices
+    //  b) Project the volume onto the film plane and calculate the 2D bounding box
+    //  c) Compute the size of the bounding box in pixels
+    //  d) Pass the bounding box to the DRR, radiograph, mask, and background renderers
+    //  e) Render the DRR, radiograph, mask, and background images
+    //  f) Use the DRR mask to "blank out" the areas of the radiograph not covered by the DRR
+    //  g) Compute the normalized cross correlation (NCC) between the radiograph and DRR
+    //  h) Compute (1.0 - NCC) value, this turns the problem from a maximization to a minimization one
```

Comment:
Could you add `// (1a)`, `// (1b)` .. below so that we can easily "map" the each sub task ?